soc/arm/xenvm/Kconfig.soc

```diff
@@ -0,0 +1,8 @@
+# Copyright 2020 EPAM Systems
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_XENVM
+	bool "Xen virtual machine on aarch64"
+	select ARM
```

Comment:
This is not needed

Reply:
I tried to remove `select ARM` but then I'm getting the following error:

> warning: ARM64 (defined at arch/arm/Kconfig:12) has direct dependencies ARM with value n, but is currently being y-selected by the following symbols:
>  - BOARD_XENVM (defined at boards/arm/xenvm/Kconfig.board:4), with value y, direct dependencies SOC_XENVM && <choice> (value: y), and select condition SOC_XENVM && <choice> (value: y)
>  - SOC_XENVM (defined at soc/arm/xenvm/Kconfig.soc:4), with value y, direct dependencies <choice> (value: y), and select condition <choice> (value: y)